Formula/embulk.rb

```diff
@@ -8,12 +8,10 @@ class Embulk < Formula
 
   depends_on :java
 
-  skip_clean "libexec"
-
   def install
-    (libexec/"bin").install "embulk-#{version}.jar" => "embulk"
-    chmod 0755, libexec/"bin/embulk"
-    bin.write_exec_script libexec/"bin/embulk"
+    # Execute through /bin/bash to be compatible with OS X 10.9.
+    libexec.install "embulk-#{version}.jar" => "embulk.jar"
+    (bin/"embulk").write "#!/bin/bash\nexec /bin/bash \"#{libexec/"embulk.jar"}\" \"$@\"\n"
```

Comment:
Please replace this line with the following to make the code easier to read:

```ruby
(bin/"embulk").write <<-EOS.undent
  #!/bin/bash
  exec /bin/bash "#{libexec}/embulk.jar" "$@"
EOS
```

Also, would it make sense to use [`Pathname#write_jar_script`](https://github.com/Homebrew/brew/blob/6dc72f2679208f34d9d09f547dda5a2ba0714a9f/Library/Homebrew/extend/pathname.rb#L425-L432) instead? This would of course skip the Java-specific setup in your binary and leave this to us, but maybe that's fine from your perspective. Here's how that would look like:

```ruby
bin.write_jar_script libexec/"embulk.jar", "embulk"
```